herddb-core/src/main/java/herddb/model/Table.java

```diff
@@ -176,13 +179,17 @@ public static Table deserialize(byte[] data) {
             for (int i = 0; i < ncols; i++) {
                 long cversion = dii.readVLong(); // version
                 long cflags = dii.readVLong(); // flags for future implementations
-                if (cversion != 1 || cflags != 0) {
+                if (cversion != 1 || (cflags != 0 && cflags != COLUMNFLAGS_HAS_DEFAULT_VALUE)) {
                     throw new IOException("corrupted table file");
                 }
                 String cname = dii.readUTF();
                 int type = dii.readVInt();
                 int serialPosition = dii.readVInt();
-                columns[i] = Column.column(cname, type, serialPosition);
+                Bytes defaultValue = null;
+                if (cflags == 1) {
```

Comment:
Use constant COLUMNFLAGS_HAS_DEFAULT_VALUE

Reply:
done.
I am using now bitwise comparison, this way it is clearer the meaning of the flag for future extensions